sdks/v4-sdk/src/utils/encodeRouteToPath.ts

```diff
@@ -0,0 +1,16 @@
+export const encodeRouteToPath = (poolKeys: any[], currencyIn: string): any[] => {
```

Comment:
aha, I actually implemented v4 route `encodeRouteToPath` in SOR: https://github.com/Uniswap/smart-order-router/blob/main/src/providers/on-chain-quote-provider.ts#L444. Feel free to copy and paste here, so that I can leverage your work.

Reply:
and it's already working, as I wrote an quote test in alpha route integ test for a v4 pool.